src/types/tokens.rs

```diff
@@ -50,19 +94,38 @@ impl FTBalance {
         }
     }
 
+    /// Parses float string and stores the value in defined precision.
+    ///
+    /// # Examples
+    ///
+    /// ## Defining 2.5 USDT
+    /// ```rust
+    /// use near_api::USDT_BALANCE;
+    ///
+    /// let usdt_balance = USDT_BALANCE.with_float_str("2.515").unwrap();
+    ///
+    /// assert_eq!(usdt_balance.amount(), 2_515_000);
+    /// ```
     pub fn with_float_str(&self, float_str: &str) -> Result<Self, DecimalNumberParsingError> {
         crate::common::utils::parse_decimal_number(float_str, 10u128.pow(self.decimals as u32))
             .map(|amount| self.with_amount(amount))
     }
 
+    /// Returns the amount without any transformations.
+    ///
+    /// The [NearToken] alternative is [NearToken::as_yoctonear].
     pub const fn amount(&self) -> u128 {
         self.balance
     }
 
+    /// Returns the amount as a whole number in the defined precision.
+    ///
+    /// The [NearToken] alternative is [NearToken::as_near].
```

Comment:
i'd rename alternative everywhere to `analogy` or remove the references to NearToken everywhere, as they won't help understand, if a person cannot understand NearToken conversions, and the tasks are roughly the same complexity

Reply:
But there is a chance that the user has already worked with NearToken, so it might be easier to understand. Renamed to analogy